packages/firebase_ai/firebase_ai/lib/src/api.dart

```diff
@@ -1523,3 +1523,63 @@ SearchEntryPoint _parseSearchEntryPoint(Object? jsonObject) {
     renderedContent: renderedContent,
   );
 }
+
+/// Supported programming languages for the generated code.
+enum CodeLanguage {
+  /// Unspecified status. This value should not be used.
+  unspecified('LANGUAGE_UNSPECIFIED'),
+
+  /// Python language.
+  python('PYTHON');
+
+  const CodeLanguage(this._jsonString);
+
+  final String _jsonString;
+
+  /// Convert to json format.
+  String toJson() => _jsonString;
+
+  /// Parse the json string to [CodeLanguage].
+  static CodeLanguage parseValue(String jsonObject) {
+    return switch (jsonObject) {
+      'LANGUAGE_UNSPECIFIED' => CodeLanguage.unspecified,
+      'PYTHON' => CodeLanguage.python,
+      _ => throw FormatException('Unhandled Outcome format', jsonObject),
```

Comment:
Update the string to Language.  Also, do you want to throw an exception if it uses a different language, or just return Unspecified?  (In Unity I just convert it to Unspecified, since I didn't want to have problems if the backend suddenly did something different, but I'm not sure what the others did.)